packages/@aws-cdk/aws-chatbot/test/slack-channel-configuration.test.ts

```diff
@@ -118,6 +118,16 @@ describe('SlackChannelConfiguration', () => {
     });
   });
 
+  test('allows adding a Topic to the existing SlackChannel', () => {
+    const slackChannelFromArn = chatbot.SlackChannelConfiguration.fromSlackChannelConfigurationArn(stack, 'RetrievedSlackChannel', 'arn:aws:chatbot::1234567890:chat-configuration/slack-channel/my-slack');
+
+    const topic = new sns.Topic(stack, 'MyTopic');
+
+    slackChannelFromArn.addNotificationTopic(topic);
+
+    Template.fromStack(stack).resourceCountIs('AWS::Chatbot::SlackChannelConfiguration', 0);
```

Comment:
Is this correct? Shouldn't you check that the SnsTopic is there in the stack? Something like the test above?

Reply:
thanks, updated now

Reply:
Perhaps my coffee has not kicked in yet, but how is
`Template.fromStack(stack).resourceCountIs('AWS::Chatbot::SlackChannelConfiguration', 0);`
relevant to the changes/ test?